.gitignore

```diff
@@ -21,7 +21,6 @@ TestResults/
 onnxruntime.egg-info
 nuget_root/
 .packages/
-.vscode/
```

Comment:
It might be better to keep .vscode/ to .gitignore. The directory changes frequently (like debugging setting), but not needed to submit in most cases.

Reply:
Is there a way to only keep .vscode/settings.json? I tried `!.vscode/settings.json` but it didn't work for me

Reply:
The following seems working in my machine:
```
.vscode
!/.vscode/settings.json
```

Reply:
Thanks! Done.